openmls/src/group/tests/external_remove_proposal.rs

```diff
@@ -126,7 +127,11 @@ fn external_remove_proposal_should_remove_member(
     // get Bob's index
     let bob_index = alice_group
         .members()
-        .find(|member| member.credential.identity() == b"Bob")
+        .find(|member| {
+            let identity =
+                VLBytes::tls_deserialize_exact(member.credential.serialized_content()).unwrap();
+            identity.as_slice() == b"Bob"
+        })
```

Comment:
This could be a `member.credential.deserialized()`, but since this all tests it not worh rewriting everything.
But the function on `BasicCredential` was actually broken. I changed the test here to use it and fixed it.
I don't think it's worth rewriting all tests like this (also it's slower).